jetstream/first-time-wallpaper-newtab.toml

```diff
@@ -15,3 +15,45 @@ select_expression = 'SUM(0)'
 select_expression = 'SUM(0)'
 [metrics.new_tab_enabled_on_new_tabs]
 select_expression = 'SUM(0)'
+
+
+[data_sources.main]
+from_expression = """(
+    SELECT
+        client_id,
+        DATE(submission_timestamp) AS submission_date
+    FROM `moz-fx-data-shared-prod.telemetry_stable.main_v5`
+    WHERE DATE(submission_timestamp) = '2025-03-04'
+)"""
+experiments_column_type = "none"
+friendly_name = "Main no-op"
+description = "Main ping table without meaningful metrics"
+
+[data_sources.as_sessions]
+from_expression = """(
+    SELECT
+        client_id,
+        DATE(submission_timestamp) AS submission_date
+    FROM mozdata.activity_stream.sessions
+        WHERE DATE(submission_timestamp) = '2025-03-04'
+    )"""
+experiments_column_type = "none"
+friendly_name = "Activity Stream Sessions no-op"
+description = "Activity Stream Sessions without meaningful metrics"
+
+[data_sources.newtab_visits_topsite_tile_interactions]
```

Comment:
This table isn't a no-op but you added a channel filter, is that right?

Reply:
Yes, that's correct. That one is still used for actual metrics.